### PR TITLE
Add Flambda2 case to Cmx_format.export_info

### PR DESCRIPTION
**Pull request by mshinwell**
Straightforward.

### CODE REVIEW

File: file_formats/cmx_format.mli [hunk 1]
```diff
@@ -33,7 +33,8 @@ open Misc
 
 type export_info =
   | Clambda of Clambda.value_approximation
-  | Flambda of Export_info.t
+  | Flambda1 of Export_info.t
```
Comment (gretay-js) [inline file_formats/cmx_format.mli:36]: Is there a particular reason not to keep it as Flambda?

**Review by gretay-js — APPROVED** [review]
The change looks fine to me assuming that the CI is fixed. Just one small question below.

File: file_formats/cmx_format.mli (hunk 1 above)
Comment (mshinwell) [reply file_formats/cmx_format.mli:36]: Just to try to avoid confusion, since the type is being changed anyway.

**Comment by mshinwell** [comment]
This needs https://github.com/ocaml-flambda/flambda-backend/pull/95 first.

**Pull request merged**